manifests/unit_file.pp

```diff
@@ -93,7 +93,10 @@
   }
 
   if $_target {
-    $_ensure = 'link'
+    $_ensure = $ensure ? {
+      'absent' => 'absent',
+      default  => 'link',
+    }
```

Comment:
There is a [stdlib function](https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/functions/ensure.pp) for this:

```suggestion
    $_ensure = stdlib::ensure($ensure, link)
```

Reply:
The stdlib requirement in `metadata.json` would also need to be set to minimum of 6.6.0 to use `stdlib::ensure()`.

Reply:
Changed, thanks! :)